README.md

```diff
@@ -65,7 +65,23 @@ Add-on for the SpatialLite library that adds support for OpenStreetMap data form
 * Do you have a question? ... [ask on StackOverflow](https://stackoverflow.com/questions/ask?tags=spatiallite)
 * Do you want to contribute piece of code? ... [submit a pull-request](https://github.com/lukaskabrt/spatiallite-net/pulls)
     * `master` branch contains the code being worked on
+    * `develop` branch requires passing CI checks before merging PRs
 
+## Development
```

Comment:
Information about branch protection shouldn't be in README file. Just add summary as a comment to the issue.